src/main/java/core/basesyntax/Zoo.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Zoo {
+    public static List<Animal> aquariumAnimals = new ArrayList();
+    public static List<Animal> enclosureAnimals = new ArrayList();
+    public static List<Animal> aviaryAnimals = new ArrayList();
```

Comment:
let's create one list which parameterized with Animal( to call method eat() for example) and lists for each interface to call theirs methods(swim() , run(), fly()..)